saleor/order/actions.py

```diff
@@ -120,7 +120,7 @@ def order_created(
         )
 
     channel = order_info.channel
-    if channel.automatically_confirm_all_new_orders or from_draft:
+    if channel.automatically_confirm_all_new_orders and from_draft:
```

Comment:
does this mean that we only confirm the order when the flag is enabled AND it is a draft? shouldn't be: `if channel.automatically_confirm_all_new_orders:` ? to confirm in both cases (regular order and draft then the flag is enabled)?